bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/KontaktBlatt.java

```diff
@@ -424,14 +437,20 @@ private void addListener(Kontakt kontakt) {
 		try {
 			boolean mandatorEditGuard = kontakt.istMandant();
 
+			List<ISticker> lSticker = StickerServiceHolder.get().getStickers(kontakt.toIContact().asIPatient());
+			boolean hasSticker = lSticker.stream().anyMatch(sticker -> sticker.getId().equals("activate_mediorder"));
```

Comment:
Wir haben jetzt schon an ein paar Stellen den literal String `activate_mediorder` - wir kopieren den immer, das gefällt mir nicht. Hast du einen Vorschlag wie und wo wir den als Konstante in einer Klasse ablegen können? 
Dann kannst du das dort dokumentieren, und es ist auch leicht referenzen darauf zu finden?!